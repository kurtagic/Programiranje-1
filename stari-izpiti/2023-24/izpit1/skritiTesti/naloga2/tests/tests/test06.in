1 38 208 607
12 2 531
12 2 154
12 2 308
12 2 476
12 2 113
12 2 234
12 2 364
12 2 539
12 2 43
12 2 343
10 2 60
12 2 636
10 68 594
10 2 396
12 68 97
12 2 445
16 68 538
10 68 125
10 2 251
16 68 223
16 2 104
20 149 411
14 48 660
16 68 387
16 48 595
10 2 166
12 149 556
20 2 87
16 68 701
16 149 318
10 68 407
14 149 542
10 48 620
14 149 671
14 2 264
16 149 212
14 149 136
20 68 342
16 149 107
14 48 507
14 2 647
16 149 141
20 68 77
14 68 322
16 68 218
12 68 282
10 149 464
10 48 7
14 2 626
16 2 24
20 9 424
10 48 380
16 149 32
20 89 229
35 89 116
20 149 360
35 2 453
35 48 519
16 149 261
16 9 269
16 48 560
12 149 498
23 66 527
12 163 114
16 87 587
23 9 611
14 149 105
20 87 452
24 89 664
16 68 546
12 9 709
16 68 243
23 149 270
24 163 334
10 9 449
4 68 96
16 48 308
4 66 262
10 163 322
16 66 652
17 66 16
20 2 641
16 9 653
13 2 236
14 163 558
14 9 223
13 87 179
4 48 438
23 2 326
13 9 370
36 48 511
12 149 211
12 2 175
23 2 314
24 163 435
14 87 662
33 68 645
24 89 676
35 2 297
13 89 663
10 68 211
17 66 201
12 48 198
24 125 659
10 89 644
14 149 559
13 141 384
12 68 195
23 87 253
33 48 24
13 87 519
13 2 239
35 149 248
14 141 518
23 141 584
24 118 591
14 87 228
24 118 395
16 2 591
16 68 677
24 9 613
4 87 548
14 89 329
17 48 323
4 136 262
20 48 693
36 125 661
14 182 638
24 87 497
17 118 64
17 141 542
14 118 298
23 9 638
35 163 639
36 182 709
26 48 455
17 89 209
4 87 298
14 48 339
13 149 371
34 9 521
20 141 610
24 87 8
10 68 403
4 118 307
33 47 619
4 202 513
34 66 705
34 125 110
23 182 158
12 202 48
35 141 14
20 47 527
33 66 6
26 202 525
33 125 98
16 9 23
21 163 653
34 141 307
10 118 311
33 47 272
13 125 460
36 202 632
10 68 132
23 2 137
24 118 34
35 182 608
23 48 484
26 68 72
14 2 651
14 66 597
36 125 218
13 141 266
21 136 456
14 9 285
21 125 541
14 125 126
13 118 152
33 9 83
16 47 478
14 87 244
16 9 557
14 182 403
13 149 53
12 89 469
10 118 14
16 68 394
4 149 368
26 149 281
23 182 670
36 163 571
24 87 673
13 89 490
24 136 632
24 9 567
24 149 275
10 9 280
34 149 138
10 163 407
14 136 470
16 87 146
20 9 628
24 87 156
14 141 519
35 68 136
17 68 430
35 202 347
10 136 444
4 141 674
34 12 64
26 70 48
26 202 327
13 2 468
36 202 210
10 141 278
13 70 339
17 108 133
13 125 113
4 136 219
3 202 571
14 48 471
4 48 368
36 163 703
22 2 602
14 108 466
20 136 280
4 66 572
33 9 454
23 157 424
12 202 167
3 124 112
12 9 486
10 12 97
24 68 574
13 66 557
20 118 653
14 141 253
14 149 709
23 47 638
35 47 383
33 136 230
16 47 249
29 89 64
33 124 241
10 163 235
14 136 269
17 12 594
13 202 192
36 47 228
21 157 449
24 9 681
23 89 174
21 125 630
4 171 195
17 47 361
26 2 52
22 118 74
21 125 223
21 41 161
0 182 674
3 118 34
20 9 694
10 182 630
23 41 567
20 68 617
10 66 662
10 157 144
21 125 288
33 163 72
22 108 639
24 9 181
20 124 432
17 68 192
21 124 528
28 118 653
21 163 322
16 48 129
21 141 307
21 118 306
16 171 441
10 31 206
6 167 303
14 157 306
19 118 549
10 108 328
16 108 61
20 171 545
22 89 642
14 68 49
9 125 237
4 47 313
33 141 616
33 68 507
4 202 324
4 149 551
14 47 329
21 12 194
21 124 194
19 118 499
3 66 73
6 25 350
10 157 446
5 66 171
9 157 548
24 202 423
10 171 601
33 149 491
19 125 382
6 70 414
0 157 600
26 125 153
19 70 542
24 163 520
26 157 190
33 108 428
17 113 427
19 182 10
10 182 704
23 171 594
14 19 307
12 113 520
17 31 365
24 108 160
19 2 160
4 202 367
21 91 227
28 182 327
6 202 333
14 9 339
14 31 524
20 113 500
12 31 43
10 167 643
21 202 94
29 68 262
13 19 275
5 171 228
21 41 565
14 31 661
26 70 396
20 47 651
14 118 192
10 108 576
10 35 602
14 70 683
14 113 475
24 47 168
14 31 484
14 163 245
16 70 187
21 12 514
21 25 142
9 9 457
36 47 664
4 59 373
16 33 710
23 108 574
33 157 341
20 167 404
16 108 438
4 82 590
16 12 314
20 25 470
20 70 551
21 79 562
4 171 112
4 193 579
23 182 250
21 35 611
23 68 462
24 31 258
36 2 140
24 203 20
17 124 505
9 74 589
34 45 338
21 82 452
35 19 227
4 68 259
4 2 344
36 33 347
4 12 362
4 33 554
14 163 279
14 82 415
16 19 326
33 12 635
35 33 129
23 9 269
3 108 250
17 167 414
24 41 439
10 167 148
19 82 101
33 70 257
26 79 465
28 79 271
24 48 452
16 70 320
20 74 551
5 25 362
6 89 521
35 167 548
20 171 117
21 89 135
33 149 135
9 48 299
16 124 315
22 48 380
13 82 349
24 91 632
14 193 11
14 31 193
3 59 95
3 171 296
20 113 689
0 59 239
16 31 485
4 124 609
35 31 171
10 41 109
12 113 340
3 70 254
13 35 81
0 74 411
24 136 446
16 118 254
20 134 108
20 124 311
4 66 129
35 45 40
16 182 402
14 125 218
22 35 316
17 193 300
4 25 140
34 87 711
22 129 42
23 108 506
24 74 636
35 48 334
17 82 681
24 48 78
9 134 63
4 45 561
10 12 468
22 78 385
24 108 525
16 132 305
4 66 428
33 107 372
23 144 243
24 163 515
24 163 157
17 182 124
0 41 197
16 202 194
15 89 46
22 47 354
10 129 592
14 59 2
14 53 672
35 74 130
35 25 639
16 91 331
24 31 465
14 41 121
14 41 198
14 79 69
20 107 627
26 157 47
29 203 135
4 108 43
12 91 459
17 167 242
0 25 394
24 125 383
16 141 349
12 79 571
34 19 529
17 45 98
6 25 174
35 124 669
24 66 134
9 12 13
35 144 682
14 163 115
21 202 337
17 141 134
17 2 402
35 19 200
12 157 213
24 182 251
10 45 241
4 70 367
26 118 167
10 163 252
29 97 647
20 157 488
26 125 55
4 182 598
10 41 127
33 136 1
14 2 451
14 61 685
5 202 41
4 129 408
4 182 176
14 48 401
0 118 434
14 41 291
10 9 259
5 205 33
34 59 415
32 205 665
16 129 77
16 108 676
14 45 461
10 113 79
4 193 56
3 61 237
15 9 447
10 59 706
0 157 280
12 136 679
3 171 518
26 68 599
10 129 324
10 149 460
15 9 591
20 38 529
36 41 478
14 134 91
24 74 252
22 31 46
3 2 64
10 124 678
10 202 169
33 41 39
24 144 250
15 91 499
3 125 573
21 154 506
21 202 434
22 31 90
3 35 37
15 91 23
35 203 486
12 202 540
22 124 281
10 53 498
14 89 572
4 35 589
23 74 655
16 134 514
4 154 58
17 78 196
22 182 485
3 78 189
5 74 17
4 45 254
10 108 697
4 61 335
14 118 545
10 149 320
3 202 344
24 2 548
5 203 334
22 182 392
14 132 234
22 108 488
29 97 632
36 19 226
8 70 333
23 87 482
4 108 647
9 61 75
3 43 297
22 41 62
23 2 182
15 125 424
19 143 18
16 83 275
3 171 537
6 182 65
13 171 386
12 69 654
35 182 286
3 186 285
4 66 381
14 118 172
6 31 407
22 83 356
14 25 539
20 124 13
21 19 63
9 31 14
16 139 482
4 182 291
4 182 516
21 143 310
10 91 82
12 139 658
14 157 591
10 61 628
17 61 172
14 25 271
