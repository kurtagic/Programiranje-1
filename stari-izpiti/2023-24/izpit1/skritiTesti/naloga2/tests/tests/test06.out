[0, 0, 651, 0, 0, 0, 0, 0, 0, 709, 0, 0, 635, 0, 0, 0, 0, 0, 0, 529, 0, 0, 0, 0, 0, 639, 0, 0, 0, 0, 0, 661, 0, 710, 0, 611, 0, 0, 529, 0, 0, 567, 0, 297, 0, 561, 0, 664, 693, 0, 0, 0, 0, 672, 0, 0, 0, 0, 0, 706, 0, 685, 0, 0, 0, 0, 705, 0, 701, 654, 683, 0, 0, 0, 655, 0, 0, 0, 385, 571, 0, 0, 681, 356, 0, 0, 0, 711, 0, 676, 0, 632, 0, 0, 0, 0, 0, 647, 0, 0, 0, 0, 0, 0, 0, 0, 0, 627, 697, 0, 0, 0, 0, 689, 0, 0, 0, 0, 653, 0, 0, 0, 0, 0, 678, 661, 0, 0, 0, 592, 0, 0, 305, 0, 514, 0, 679, 0, 0, 658, 0, 674, 0, 310, 682, 0, 0, 0, 0, 709, 0, 0, 0, 0, 506, 0, 0, 600, 0, 0, 0, 0, 0, 703, 0, 0, 0, 643, 0, 0, 0, 601, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 709, 0, 0, 0, 285, 0, 0, 0, 0, 0, 0, 579, 0, 0, 0, 0, 0, 0, 0, 0, 632, 486, 0, 665, 0, 0]
